src/index.js

```diff
@@ -14,9 +15,7 @@ log.info('🗿 npm ↔️ Algolia replication starts ⛷ 🐌 🛰');
 
 let loopStart = Date.now();
 
-const { index: mainIndex, client } = createAlgoliaIndex(config.indexName);
-const { index: bootstrapIndex } = createAlgoliaIndex(config.bootstrapIndexName);
-const stateManager = createStateManager(mainIndex);
+let stateManager;
```

Comment:
is this just to have the variable global? I'd rather pass it around than have a mutable (maybe undefined) variable :)

Reply:
It's just to have the variable global for the moment, I'll change that in an other PR. 
I can do it now if you want

Reply:
would make sense here to me :)

Reply:
Did it, plus added some test 🙆‍♂

Reply:
3f6dfb1